docs/integrations/sources/salesforce.md

```diff
@@ -132,20 +166,28 @@ Salesforce allows extracting data using either the [BULK API](https://developer.
 
 More information on the differences between various Salesforce APIs can be found [here](https://help.salesforce.com/s/articleView?id=sf.integrate_what_is_api.htm&type=5).
 
-:::info Force Using Bulk API
+:::note Force Using Bulk API
```

Comment:
this is rendering weirdly

Reply:
> this is rendering weirdly

@katmarkham Is this happening via Docusaurus or in the markdown preview in GitHub? These are [Docusaurus admonitions](https://docusaurus.io/docs/markdown-features/admonitions) so I don't think they'll render in GitHub markdown.

This is what I see when running Docusaurus locally:

<img width="713" alt="Screenshot 2023-11-10 at 10 40 48 AM" src="https://github.com/airbytehq/airbyte/assets/6109223/892b8326-4cc9-47b6-bc8b-b4f7192c9386">